ui/admin/app/components/form/onboarding/choose-path/index.hbs

```diff
@@ -0,0 +1,34 @@
+<Rose::Form
+  class='full-width'
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @onSubmit={{fn @submit this.path}}
+  as |form|
+>
+  <form.radioGroup
+    @name='paths'
+    @selectedValue={{this.path}}
+    @changed={{this.toggleRadio}}
+    as |radioGroup|
+  >
+    <div class='choose-path-cards'>
```

Comment:
do we need this wrapper `<div class='choose-path-cards'>` here?

Reply:
hmm, the space between the `radioCards` appear larger than the wireframe

Reply:
Yes, I had some issues with changing the width of the rose component and it causing the radio cards to stack on top of each other.

Reply:
Fixed. I think the `space-between` style changes allowed me to adjust the width a smidge smaller without moving the cards on top of each other.